app/build.gradle.kts

```diff
@@ -1,6 +1,7 @@
 plugins {
     id("com.android.application")
     id("org.jetbrains.kotlin.android")
+    id("kotlin-kapt")
 }
```

Comment:
Currently causing merge issues with main here. is just having `kotlin(kapt)` acceptable?

Reply:
Yes! That works also